src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/integration/PrisonApiMockServer.kt

```diff
@@ -330,6 +368,25 @@ class PrisonApiMockServer : WireMockServer(9005) {
         )
     )
   }
+  fun stubTransferInOffenderFailsNoCapacity(offenderNo: String) {
+    stubFor(
+      put("/api/offenders/$offenderNo/transfer-in")
+        .willReturn(
+          aResponse()
+            .withHeader("Content-Type", MediaType.APPLICATION_JSON_VALUE)
+            .withStatus(409)
+            .withBody(
+              """
+             {
+    "status": 409,
+    "userMessage": "The cell MDI-RECP does not have any available capacity",
+    "developerMessage": "The cell MDI-RECP does not have any available capacity"
+}
+            """
+            )
+        )
+    )
+  }
```

Comment:
Transfer-in return the same error when reception is full. I added this to mock server to save time in future (if we decide to implement specific error message for this scenario as well.)